using-mobile-extensions/adobe-target/README.md

```diff
@@ -84,271 +98,153 @@ public class TargetApp extends Application {
 {% tab title="iOS" %}
 #### Objective-C
 
-1. In your app's `didFinishLaunchingWithOptions` function register the Target extension
+1. In your app's `didFinishLaunchingWithOptions` function register the Target extension with Mobile Core:
 
 ```objectivec
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [ACPCore configureWithAppId:@"yourAppId"];
   [ACPIdentity registerExtension];
   [ACPTarget registerExtension];
-
+  [ACPCore start:nil];
   // Override point for customization after application launch.
   return YES;
 }
 ```
 
 #### Swift
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  ACPCore.configure(withAppId: "yourAppId")   
+  ACPTarget.registerExtension()
+  ACPIdentity.registerExtension()
+  ACPCore.start(nil)
+  // Override point for customization after application launch. 
+  return true;
+}
+```
+
 {% endtab %}
 {% endtabs %}
 
-## Prefetch offers <a id="integrating-adobe-target-with-analytics-a-4-t"></a>
+## Parameters in a Target request
 
-The SDK can minimize the number of times it reaches out to Target servers to fetch offers by caching server responses. When this feature is enabled, offer content is retrieved and cached during the prefetch call. This content is retrieved from the cache for all future calls that contain cached content for the specified mbox name. This prefetch process reduces offer load time, network calls made to Target servers, and allows Target to be notified which mbox was visited by the mobile app user.
+### Target Order
 
-{% hint style="warning" %}
-Prefetched offer content does not persist across launches. The prefetch content is cached as long as the application lives or until the API that is used to clear the cache is called. For more information, see [Clear prefetch offer cache](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target#clear-prefetch-offer-cache).
-{% endhint %}
-
-### Using the prefetch APIs
+The `TargetOrder` class encapsulates the order ID, the order total and the purchased product IDs, and you can instantiate this class to create order parameters. For more information on Target Order parameters, see [here](https://docs.adobe.com/content/help/en/target/using/implement-target/client-side/mbox-implement/orderconfirm-create.html)
```

Comment:
- Please reword as follows:
"The `TargetOrder` class encapsulates the order ID, the order total, and the purchased product IDs. You can instantiate this class to create order parameters. 
- In the "For more information..." sentence, please replace "on" with "about". 
- Please replace "here" with the actual name of the topic --> [Create an Order Confirmation mbox - mbox.js] and add a period at the end of the sentence.